api/proto/info/info.proto

```diff
@@ -507,6 +509,40 @@ message ZInfoNetworkInstance {
   repeated ErrorInfo networkErr = 40;
 }
 
+message UsageInfo {
+  google.protobuf.Timestamp createTime = 1;
+  uint32 refCount = 2;
+  google.protobuf.Timestamp lastRefcountChangeTime = 3; // Last time when refCount changed
+}
+
+message VolumeResources {
+  uint64 maxSizeBytes = 1; // From config
+  uint64 curSizeBytes = 2; // Current disk usage
+}
+
+message ZInfoVolume {
+  string uuid = 1;
+  string displayName = 2;
+  UsageInfo usage = 3;
+  VolumeResources resources = 4;
+  ZSwState state = 5;
+  uint32 downloadProgress = 6;
+
+  ErrorInfo volumeErr = 7;
+}
+
+message ContentResources {
+  uint64 curSizeBytes = 1;  // Current disk usage
+}
+
+message ZInfoContentTree {
```

Comment:
Since we potentially end up with multiple different hashes for the same contentTree uuid, I think we need to add a sha256 to this object. That isn't in the document.

Reply:
will add sha256 field to ZInfoContentTree.

Reply:
Great.